homeassistant/components/cover/tellduslive.py

```diff
@@ -33,14 +33,11 @@ def is_closed(self):
     def close_cover(self, **kwargs):
         """Close the cover."""
         self.device.down()
-        self.changed()
 
     def open_cover(self, **kwargs):
         """Open the cover."""
         self.device.up()
-        self.changed()
 
     def stop_cover(self, **kwargs):
         """Stop the cover."""
         self.device.stop()
-        self.changed()
```

Comment:
@MartinHjelmare this is not necessary anymore, if I understand you correctly here: https://github.com/home-assistant/home-assistant/pull/18438#discussion_r233166260.